LoopKit.xcodeproj/project.pbxproj

```diff
@@ -2461,11 +2463,9 @@
 				B4C004B8241085FE00B40429 /* Color.swift */,
 				B4F3D24E24AF59B50095CE44 /* DeviceLifecycleProgressState+Color.swift */,
 				B46A70B924B3B4BF00B4DEEC /* DeviceStatusHighlight+Color.swift */,
-				E96175AD24B7BE38008E5080 /* Dictionary.swift */,
 				89BE75C62464B4A900B145D9 /* Environment+Dismiss.swift */,
 				B429D66B24BF7204003E1B4A /* GlucoseTrend.swift */,
 				B42C950824A3BCC200857C73 /* GlucoseValueType+Color.swift */,
-				89186C0624BF7FC70003D0F3 /* Guardrail+UI.swift */,
```

Comment:
Zero clue what these are showing up as deletions - I force-checked-out the `project.pbxproj` file and they're still showing up this way

Reply:
Weird. And in that extensions folder in Xcode, the files still appear normally?

Reply:
Yes! That's the really interesting part.

Reply:
Sometimes Xcode "cleans up" project files just by opening them in Xcode. Could be leftover cruft from a previous merge.